### PR TITLE
Adding the wrapperComponent prop to RemoteFramesProvider

### PR DESCRIPTION
**Pull request by wistcc**
We now are able to wrap the `GlobalTarget` inside remote-frames with any HOC we would need to.

### CODE REVIEW

File: src/RemoteFramesProvider.js [hunk 1]
```diff
@@ -65,14 +65,29 @@ class RemoteFramesProvider extends Component {
...
-      targetDomElement
+      />
     )
+
+    const WrapperComponent = this.props.wrapperComponent
+
+    if (WrapperComponent) {
+      render(
+        <WrapperComponent>
+          <Target />
```
Comment (Deschtex) [inline src/RemoteFramesProvider.js]: Shouldn't this be `{Target}` instead, considering `Target` is not a component function?

File: src/RemoteFramesProvider.js [hunk 2]
```diff
@@ -65,14 +65,29 @@ class RemoteFramesProvider extends Component {
...
+        </WrapperComponent>,
+        targetDomElement
+      )
+    } else {
+      render(
+        <Target />,
```
Comment (Deschtex) [inline src/RemoteFramesProvider.js]: Same here.

File: src/RemoteFramesProvider.js (hunk 1 above)
Comment (wistcc) [reply src/RemoteFramesProvider.js]: Ha! You are right, good catch.

**Review by wovalle — APPROVED** [review]
LGTM!

**Pull request merged**